src/web/lib/braze/checkBrazeDependencies.test.ts

```diff
@@ -0,0 +1,242 @@
+import { checkBrazeDependencies } from './checkBrazeDependencies';
+
+afterEach(() => {
+	// Wait for any unsettled promises to complete at the end of each test. Once
+	// we encounter a failure in our list of checks we don't need to wait on
+	// subsequent operations to complete which is why there might be unsettled
+	// promises.
+	const flushPromises = new Promise(setImmediate);
+	return flushPromises;
+});
+
+let mockBrazeUuid: string | null;
```

Comment:
I don't ❤️  all the mocking we're doing here but I think it's worth it for the confidence that the wiring etc is all correct, and at least it's all confined to this one test. This is much more of an integration test than a unit test.